pages/common/npm.md

```diff
@@ -2,6 +2,7 @@
 
 > JavaScript and Node.js package manager.
 > Manage Node.js projects and their module dependencies.
+> Homepage: <http://npm.org/>.
```

Comment:
Should this be https://www.npmjs.com/ or https://www.npmjs.com/package/npm?

Reply:
Yeah, definitely npmjs.org 👍

Reply:
As far as I'm aware, the domain is actually npmjs.org. As least, that's what I've always gone to.

Reply:
> As far as I'm aware, the domain is actually npmjs.org.

That's exactly what @MasterOdin is saying :)
Note that you wrote https://npm.org/ instead (without the "js"), which is the website for the National Association of Pastoral Musicians (of the US).